fedn/network/api/interface.py

```diff
@@ -1002,13 +1002,17 @@ def start_session(
             return jsonify({"success": False, "message": "A session is already running."})
 
         # Check if compute package is set
-        if not self.statestore.get_compute_package():
+        package = self.statestore.get_compute_package()
+        if not package:
             return jsonify(
                 {
                     "success": False,
                     "message": "No compute package set. Set compute package before starting session.",
                 }
             )
+        if not helper:
+            # get helper from compute package
+            helper = package["helper"]
```

Comment:
Wait, it we can do this here, we should be able to do it also on the combiner? In that case - can we completely eliminate the need to set the helper in the session_config and instead have it encoded in the fedn.yaml?  Even if we in the future will not always have a package, I think we will always have the concept of a Project where a fedn.yaml is used to configure things.

Reply:
I do not completely follow, the package is not used by combiner, sire you could fetch this iformation using the statestore client. Also, this is beyond this PR. This is a small fix.